### PR TITLE
Focus color of buttons should be more saturated/darker (Issue #2134)

### PR DESCRIPTION
**Pull request by CaveSpectre11**
Issue #2134 reports an issue with hovering or tabbing focus onto a button lightens the button rather than darkens it, making it more apt to appear disabled.

This PR corrects this behavior by changing the derived color on hover and focused to -30% rather than +30%.

One potential remaining issue is the action that occurs when the button is focused upon tabbing; there is an animated effect that dims the button.  While it is now darker than previous, it still becomes somewhat washed out and not as vivid as a mouse hover.

### CODE REVIEW

**Comment by CaveSpectre11** [comment]
Hover (right side is hovered):
screenshot from 2018-12-31 12-40-10

screenshot from 2018-12-31 12-40-45

Focused (right side is focused)
screenshot from 2018-12-31 12-45-06

screenshot from 2018-12-31 12-44-28

**Comment by ManfredKarrer** [comment]
@ripcurlx Leave it to you. Maybe @pedromvpg could add his opinion as well!

**Review by ripcurlx — APPROVED** [review]
utACK from a code perspective, but I'd love to get an ACK from @pedromvpg as well, if he's fine with these UI changes.

**Comment by ripcurlx** [comment]
ACK - just tested it locally.  @pedromvpg If you want to change the color/saturation, we can adapt it in the next release again.

**Pull request merged**
